ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -10,6 +10,70 @@
     handle_numpy_out,
 )
 
+# global
+import numpy as np
+from typing import Optional, Sequence, Union
+import ivy
+from ivy.func_wrapper import with_unsupported_dtypes
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    handle_numpy_dtype,
+    from_zero_dim_arrays_to_scalar,
+    handle_numpy_out,
+)
+
+def nanmedian(
+    a,
+    /,
+     *,
+    axis=None,
+    keepdims=False,
+    out=None,
+    overwrite_input=False
+):
+    is_nan = ivy.isnan(a)
+    axis = tuple(axis) if isinstance(axis, list) else axis
+
+    if not ivy.any(is_nan):
+        ret = ivy.median(a, axis=axis, keepdims=keepdims, out=out, overwrite_input=overwrite_input)
+    else:
+        a = [i for i in a if ivy.isnan(i) is False]
+        ret = ivy.median(a, axis=axis, keepdims=keepdims, out=out, overwrite_input=overwrite_input)
+
+    return ret
+# def nanmedian(
+#     a,
+#     /,
+#     *,
+#     axis=None,
+#     keepdims=False,
+#     out=None,
+#     dtype=None,
+#     where=True,
+# ):
+#     is_nan = ivy.isnan(a)
+#     axis = tuple(axis) if isinstance(axis, list) else axis
+
+#     if not any(is_nan):
+#         if dtype:
+#             a = ivy.astype(ivy.array(a), ivy.as_ivy_dtype(dtype))
+#         ret = ivy.median(a, axis=axis, keepdims=keepdims, out=out)
+
+#         if ivy.is_array(where):
+#             ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
+#     else:
+#         a = [i for i in a if ivy.isnan(i) == False]
+
+#         if dtype:
+#             a = ivy.astype(ivy.array(a), ivy.as_ivy_dtype(dtype))
+#         ret = ivy.median(a, axis=axis, keepdims=keepdims, out=out)
+
+#         if ivy.is_array(where):
+#             ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
+#     return ret
+
+
+nanmedian.support_native_out = True
 
```

Comment:
there should be two empty lines between this line and the next part

Reply:
Done